### DISCUSSION TITLE
Limiting buying

### DISCUSSION THREAD

**Discussion by FetFingers** [ChestShop-authors/ChestShop-3#482] [Ideas]
Hey there,
I have been trying your plugin ChestShop 3 and it works pretty good.

I do have one question tho, I was thinking on doing a kind of "black market" where few items are available from the admin shop.
But even if I have only 2 items inside the chest, the same player can buy both of them. 

I was wondering if there's any way to limit the buying process to "1 per player", where any player can only buy 1 item from that chest.

Thank you for your time!

**Comment by Phoenix616** [comment]
That's not possible within ChestShop itself (as it doesn't store purchases at all) and I don't know of any addon plugin which adds such a functionality but it would be pretty easy for an addon plugin to add such functionality by listening to ChestShop's events and tracking purchases.

**Comment by FetFingers** [comment]
So, in a way that would be like using ChestShopNotifier, but instead of
just registering transactions, also limit the time a purchase can happen!?



Max Lee ***@***.***> escreveu no dia terça, 12/10/2021 à(s)
17:03:

**Reply by Phoenix616** [reply]
Yes.

**Reply by FetFingers** [reply]
Well, I wish I could code x)
